aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -203,9 +226,9 @@ The database is ready. You can start creating the ASP.NET Core web API.
 
     The preceding `BookstoreDatabaseSettings` class is used to store the *appsettings.json* file's `BookstoreDatabaseSettings` property values. The JSON and C# property names are named identically to ease the mapping process.
 
-1. Add the following code to `Startup.ConfigureServices`, before the call to `AddMvc`:
+1. Add the following highlighted code to `Startup.ConfigureServices`:
 
-    [!code-csharp[](first-mongo-app/sample/BooksApi/Startup.cs?name=snippet_ConfigureDatabaseSettings)]
+    [!code-csharp[](first-mongo-app/sample/snippets/Startup.ConfigureServices.AddDbSettings.cs?highlight=3-7)]
```

Comment:
This is a snapshot folder ("snippets")? If so, this is a departure from what we've (I've) been doing. My guidance might be dated tho, so please let me know if some changes were made for snapshot file location.

Reply:
I'll follow what you've done in the past.

Reply:
Almost. lol 

Like this with the version inside the folder.

https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/fundamentals/host/generic-host

Only the `_snapshot` has to be added. The files are stored in parallel generally. Well ... I'm trying. This has been an evolving (for the better I hope) thing. This current 2.2 round of sample updates on the tracking issue I'm trying to bring it all together.